zap/src/test/java/org/zaproxy/zap/scan/filters/GenericFilterUtilityTest.java

```diff
@@ -0,0 +1,271 @@
+/*
```

Comment:
left adding Unit Tests for Include All functionality and also some more test cases